src/test/java/org/prebid/server/bidder/ix/IxBidderTest.java

```diff
@@ -374,6 +375,36 @@ public void makeBidsShouldReturnBannerBidWithRequestImpSizeWhenBidSizeIsEmpty()
                 .containsOnly(BidderBid.of(Bid.builder().impid("123").w(300).h(200).build(), BidType.banner, "EUR"));
     }
 
+    @Test
+    public void makeBidsShouldReturnBidWithVideoExt() throws JsonProcessingException {
+        // given
+        final Video video = Video.builder().build();
+        final HttpCall<BidRequest> httpCall = givenHttpCall(
+                BidRequest.builder()
+                        .imp(singletonList(Imp.builder().id("123").video(video).build()))
+                        .build(),
+                mapper.writeValueAsString(
+                        givenBidResponse(
+                                bidBuilder -> bidBuilder
+                                        .impid("123")
+                                        .w(300).h(200)
+                                        .ext(mapper.valueToTree(ExtPrebid.of(null, ExtBidPrebidVideo.of(1, "cat")))))));
+
+        // when
+        final Result<List<BidderBid>> result = ixBidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue())
+                .containsOnly(
+                        BidderBid.of(Bid.builder()
+                                .impid("123")
+                                .w(300)
+                                .h(200)
+                                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtBidPrebidVideo.of(1, "cat"))))
```

Comment:
We don't need to check every param when it's only about VideoExt

Reply:
We should have separate tests for sizes and video logic

Reply:
Sizes removed